### PR TITLE
Fix emitHook for 2-arg calls without a callback.

### LINKED ISSUE #373: express.use is not a function
using version 4.3.3

00:03:11.325 [14:32:56] Starting 'wct:local'...
00:03:12.239 Received {"global":{"statements":100,"branches":100,"lines":100,"functions":100}}
00:03:13.468 Installing and starting Selenium server for local browsers
00:03:18.775 Selenium server running on port 44707
00:03:18.801 [14:33:04] 'wct:local' errored after 7.47 s
00:03:18.801 [14:33:04] Error: express.use is not a function

Polymer 1.6
wct 4.3.3

### PR DESCRIPTION
**Pull request by justinfagnani**
Fixes #373 and #383

Verified that the test fails on master and passes now.

 - [ ] CHANGELOG.md has been updated

### CODE REVIEW

**Comment by justinfagnani** [comment]
Updating changelog now

**Comment by justinfagnani** [comment]
`npm run build` wasn't reporting any errors :(

**Comment by justinfagnani** [comment]
@rictic fixed errors and updated build script to just use `tsc`

File: runner/context.ts [hunk 1]
```diff
@@ -101,34 +101,33 @@ export class Context extends events.EventEmitter {
...
-      }
-      const hookArgs = Array.from(arguments).slice(1, argsEnd);
-      boundHooks = hooks.map(function(hook) {
-        return hook.bind.apply(hook, [null].concat(hookArgs));
-      });
+    let done: (err?: any) => void = (_err: any) => {};
+    let argsEnd = args.length - 1;
+    if (args[argsEnd] instanceof Function) {
+      done = args[argsEnd];
+      argsEnd = argsEnd--;
```
Comment (korzio) [inline runner/context.ts]: This construction seems strange to me. It basically doesn't change argsEnd

Comment (justinfagnani) [inline runner/context.ts]: wow, you're right. I wonder where the test coverage is lacking for this.

**Pull request merged**